src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAOSupport.scala

```diff
@@ -49,7 +49,9 @@ trait ElasticSearchDAOSupport extends LazyLogging {
       case (label: String, detail: AttributeDetail) => detailFromAttribute(label, detail)
     }
     // add the magic "_suggest" property that we'll use for autocomplete
-    val props = attributeDetailMap + ((fieldSuggest, ESType.suggestField("string")))
+    val props = attributeDetailMap +
+      (fieldSuggest -> ESType.suggestField("string"),
+        fieldDiscoverableByGroups -> new ESInternalType("string"))
```

Comment:
Nitpick on the name. Do we need the full discoverable by groups? Can this just be discoverable? Not sure if we would ever need the distinction of discoverable by something other than groups.